app/src/main/res/values/strings.xml

```diff
@@ -516,6 +516,7 @@
     <string name="past_24_hours">Past 24 hours</string>
     <string name="prefer_exoplayer_ffmpeg">Prefer FFmpeg for audio playback</string>
     <string name="prefer_exoplayer_ffmpeg_content">Use FFmpeg to decode audio, even if platform codecs are available.</string>
+    <string name="lbl_video_start_delay">Video Start Delay</string>
```

Comment:
```suggestion
    <string name="lbl_video_start_delay">Video start delay</string>
```

Reply:
Also the `lbl_` prefix isn't needed

Reply:
I have updated the string name and value as requested.